packages/mobile/locales/pt-BR/fiatExchangeFlow.json

```diff
@@ -25,6 +25,7 @@
   "receiveOnAddress": "Endereço cUSD/CELO",
   "receiveWithPonto": "GCash (com Ponto)",
   "receiveWithKotani": "M-PESA (com Kotani Pay)",
+  "receiveWithBidali": "Gift Cards and Mobile Top Up",
```

Comment:
Currently waiting for the actual translation. Will add it as soon as I receive it.